ush/config_defaults.yaml

```diff
@@ -1777,23 +1767,110 @@ task_run_fcst:
+      global_albedo4.1x1.grb: '{{ workflow.FIXam }}/global_albedo4.1x1.grb'
+      global_zorclim.1x1.grb: '{{ workflow.FIXam }}/global_zorclim.1x1.grb'
+      global_tg3clim.2.6x1.5.grb: '{{ workflow.FIXam }}/global_tg3clim.2.6x1.5.grb'
+      sfc_emissivity_idx.txt: '{{ workflow.FIXam }}/global_sfc_emissivity_idx.txt'
+      solarconstant_noaa_an.txt: '{{ workflow.FIXam }}/global_solarconstant_noaa_an.txt'
+      global_o3prdlos.f77: '{{ workflow.FIXam }}/ozprdlos_2015_new_sbuvO3_tclm15_nuchem.f77'
     lateral_boundary_conditions:
       interval_hours: 1 
       offset: 3 
       path: '{{ task_run_fcst.fv3.rundir }}/INPUT/{{ nco.NET_default }}.{{ cycle }}{{ "dot_ensmem" | env  }}.gfs_bndy.tile{{ constants.TILE_RGNL }}.f{{ task_run_fcst.fv3.lateral_boundary_conditions.offset }}.nc' 
     length: 12
     model_configure:
-      base_file: '{{ user.USHdir }}/create_model_configure_file.py' 
       update_values:
-        write_dopost: .false.
+        total_member: 1
+        start_year: !int '{{ cycle.year }}'
+        start_month: !int '{{ cycle.month }}'
+        start_day: !int '{{ cycle.day }}'
+        start_hour: !int '{{ cycle.hour }}'
+        start_minute: !int '{{ cycle.minute }}'
+        start_second: !int '{{ cycle.second }}'
+        nhours_fcst: !int '{{ workflow.FCST_LEN_HRS }}'
+        fhrot: 0
+        run_continue: false
+        ens_sps: false
+        dt_atmos: ""
+        calendar: julian
+        memuse_verbose: false
+        restart_interval: 0
+        output_1st_tstep_rst: false
+        write_dopost: false
+        ideflate: 0
+        nbits: 0
+        ichunk2d: -1
+        jchunk2d: -1
+        ichunk3d: -1
+        jchunk3d: -1
+        kchunk3d: -1
+        itasks: 1
+        quilting: true
+        write_groups: !int '{{ task_run_fcst.WRTCMP_write_groups }}'
+        write_tasks_per_group: !int '{{ task_run_fcst.WRTCMP_write_tasks_per_group }}'
+        num_files: 2
+        filename_base: 'dyn phy'
+        output_file: 'netcdf netcdf'
+        output_fh: '1  -1'
+        nsout: -1
+        output_grid: '{{ task_run_fcst.WRTCMP_output_grid }}'
     namelist:
-      base_file: /path/to/base/input.nml
+      base_file: "{{ user.PARMdir }}/input.nml.FV3"
       update_values:
+        atmos_model_nml:
+          blocksize: !int '{{ task_run_fcst.BLOCKSIZE }}'
+          ccpp_suite: '{{ workflow.CCPP_PHYS_SUITE }}'
         fv_core_nml:
-          k_split: 2
-          n_split: 6
+          target_lon: !int '{{ grid_params.LON_CTR }}'
+          target_lat: !int '{{ grid_params.LAT_CTR }}'
+          nrows_blend: !int '{{ grid_params.HALO_BLEND }}'
+          stretch_fac: !int '{{ grid_params.STRETCH_FAC }}'
+          npx: !int '{{ grid_params.NX + 1 }}'
+          npy: !int '{{ grid_params.NY + 1 }}'
+          layout:
+            - !int '{{ task_run_fcst.LAYOUT_X }}'
+            - !int '{{ task_run_fcst.LAYOUT_Y }}'
+          bc_update_interval: !int '{{ task_get_extrn_lbcs.LBC_SPEC_INTVL_HRS }}'
```

Comment:
There was logic in the setup.py and/or the generate_FV3LAM_wflow.py script that set these values. Moved them here to simplify.